app-config.yaml

```diff
@@ -248,6 +248,9 @@ catalog:
     - type: file
       target: ../catalog-model/examples/acme-corp.yaml
 
+    - type: file
```

Comment:
This probably can be removed too right? We should just add it to the `sample-templates` yaml instead